src/main/flight/altitude.c

```diff
@@ -63,6 +65,7 @@ enum {
 };
 // 40hz update rate (20hz LPF on acc)
 #define BARO_UPDATE_FREQUENCY_40HZ (1000 * 25)
+#define GPS_UPDATE_FREQUENCY_5HZ (1000 * 200)
```

Comment:
Where is this used?

Reply:
Nowhere yet, I'll remove it for this PR.